Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,435 @@
+    <f:section name="Main">
+        <f:render partial="Navigation/MainNavigation.html" arguments="{mainMenu}"/>
+        <main role="main">
+            <f:render partial="Navigation/Breadcrumb.html" arguments="{breadcrumb}"/>
+            <p>...</p>
+        </main>
+    </f:section>
+
+With the following partials:
+
+..  code-block:: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Partials/Navigation/Breadcrumb.html
+
+    <f:if condition="{breadcrumb}">
+        <ol class="breadcrumb">
+            <f:for each="{breadcrumb}" as="item">
+                <li class="breadcrumb-item{f:if(condition: item.current, then: ' active')}" >
+                    <a class="breadcrumb-link" href="{item.link}" title="{item.title}">
+                        <span class="breadcrumb-text">{item.title}</span>
+                    </a>
+                </li>
+            </f:for>
+        </ol>
+    </f:if>
+
+And
+
+..  code-block:: html
+    :caption: EXT:my_sitepackage/Resources/Private/Templates/Partials/Navigation/MainNavigation.html
+
+    <ul class="nav nav-pills">
+        <f:for each="{mainMenu}" as="menuItem">
+         <li class="nav-item {f:if(condition: menuItem.hasSubpages, then: 'dropdown')}">
+            <f:if condition="{menuItem.hasSubpages}">
+               <f:then>
+                  <!-- Item has children -->
+                  <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-expanded="false">
+                     {menuItem.title}
+                  </a>
+                  <div class="dropdown-menu">
+                     <f:for each="{menuItem.children}" as="menuItemLevel2">
+                          <f:link.page pageUid="{menuItemLevel2.data.uid}"
+                                       class="dropdown-item {f:if(condition: menuItemLevel2.active, then: 'active')}">
+                             {menuItemLevel2.title}
+                          </f:link.page>
+                     </f:for>
+                  </div>
+               </f:then>
+               <f:else>
+                  <!-- Item has no children -->
+                  <f:link.page pageUid="{menuItem.data.uid}" class="nav-link {f:if(condition: menuItem.active, then:'active')}">
+                     {menuItem.title}
+                  </f:link.page>
+               </f:else>
+            </f:if>
+         </li>
+      </f:for>
+   </ul>
+
+..  _cobj-pageview-paths:
+
+paths
+-----
+
+..  confval:: paths.[priority]
+    :name: pageview-paths
+    :type: :ref:`data-type-path` with :ref:`stdWrap <stdwrap>`
+
+    Sets an array of paths for the Fluid templates, usually
+    :file:`EXT:my_extension/Resources/Private/Templates/` or a folder below that
+    path like :file:`EXT:my_extension/Resources/Private/Templates/MyPages`.
+
+    The templates are expected in a subfolder :path:`Pages` or
+    :path:`pages`.
+
+    Fluid partials are searched for in a subfolder called :path:`Partials` or
```

Comment:
```suggestion
    Fluid partials are looked up in a sub-directory called :path:`Partials` or
```

Is :path: a mapped property? Thought I only used ":file:" always and need to memorize then :)